docs/pages/application-access/guides/automation.mdx

```diff
@@ -0,0 +1,96 @@
+---
```

Comment:
I feel like this article should wait for certbot to be completed. 240 hours for a non-expiring cert is a bit too much. I feel like we promote a bad security practice here.